API Blueprint Specification.md

```diff
@@ -766,30 +766,9 @@ Defined by the `Parameters` keyword written in a Markdown list item:
 #### Description
 Discussion of URI parameters in the _scope of the parent section_.
 
-This section **must** be composed of nested list items only. This section **must not** contain any other elements. One list item per URI parameter. The nested list items subsections inherit from the [Named section](#def-named-section) and are subject to additional formatting as follows:
+This section **must** be composed of nested list items only. This section **must not** contain any other elements. Each of the list item describes a single URI parameter using the **[Markdown Syntax for Object Notation][MSON] (MSON)**.
 
-    + <parameter name> = `<default value>` (required | optional , <type>, `<example value>`) ... <description>
-
-        <additional description>
-
-        + Values
-            + `<enumeration element 1>`
-            + `<enumeration element 2>`
-            ...
-            + `<enumeration element N>`
-
-Where:
-
-* `<parameter name>` is the parameter name as written in [Resource Section](#ResourceSection)'s URI (e.g. "id").
-* `<description>` is any **optional** Markdown-formatted description of the parameter.
-* `<additional description>` is any additional **optional** Markdown-formatted [description](#SectionDescription) of the parameter.
-* `<default value>` is an **optional** default value of the parameter – a value that is used when no value is explicitly set (optional parameters only).
-* `<example value>` is an **optional** example value of the parameter (e.g. `1234`).
-* `<type>` is the **optional** parameter type as expected by the API (e.g. "number").
-* `Values` is the **optional** enumeration of possible values
-*  and `<enumeration element n>` represents an element of enumeration type.
-* `required` is the **optional** specifier of a required parameter (this is the **default**)
```

Comment:
> The only thing I can think of is to do not make the requirement flag optional – it should be always specified otherwise parser will warn.

Are you sure you are talking about the old syntax? I think what @kylef is trying to say is:

```
+ id (string)
```

In the old syntax, the parameter `id` is `required`, while in the new syntax, the parameter `id` is `optional`.

Please note that if the user adds a bit more to it like for example:

```
+ id = `1` (string)
```

or 

```
+ id: `1` (string)
```

We can easily understand which syntax he is using and can understand whether he is saying `optional` or `required`.

I think the user will not be confused when writing the blueprint. Its when the blueprint is parsed and the snowcrash is unable to decide on the old or the new syntax.